root/edit/details/EditCoverArt.js

```diff
@@ -21,20 +21,20 @@ type Props = {
   +edit: EditCoverArtEditT,
 };
 
+function displayCoverArtTypes(types: $ReadOnlyArray<CoverArtTypeT>) {
```

Comment:
I moved it because it doesn't use anything inside the scope of `EditCoverArt`, so doesn't need to be created each time the component renders.  Would you rather I move it in a separate commit?